build/docker/package.sh

```diff
@@ -38,7 +39,7 @@ done
 gzip -f datadog-dotnet-apm.tar
```

Comment:
In the above package build, does that also build new arm64 `.deb` and `.rpm` files but with the exact same name as before?

Reply:
for `.deb` and `.rpm` packages, the python tool handles the architecture automatically. The current packages pipeline produces the following artifacts: 
![image](https://user-images.githubusercontent.com/69803/118161692-28b78100-b420-11eb-8ee9-68945180309f.png)

The breaking change is on the `.tar.gz` file that I'm gonna revert in the next commit.

Reply:
breaking change fixed in 3684467